src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs

```diff
@@ -24,13 +34,16 @@
         [text-input/text-input-with-label
          {:label           (i18n/label :t/name)
           :placeholder     (i18n/label :t/specify-name)
+          :style           styles/input
           :container       styles/input-container
           :default-value   (get-in manage-mailserver [:name :value])
           :on-change-text  #(re-frame/dispatch [:mailserver-set-input :name %])
           :auto-focus      true}]
         [text-input/text-input-with-label
          {:label           (i18n/label :t/mailserver-address)
           :placeholder     (i18n/label :t/specify-mailserver-address)
+          :content         qr-code
```

Comment:
Does this has to be inside the text input?

Reply:
it needs to be a sibling, in the same way we do for `contact-code`, so I would not be able to re-use text-input-with-label

Reply:
Can't you wrap in a view both qr-code and input? Or maybe I don't quite see what the result is.
From a semantic perspective I would not expect an input to be a wrapper for a QR code.

Reply:
Basically,  `text-input-with-label` put things like (think of brackets as flex boxes):
```
[:label
 :text-input]
```

I need them to be:
```
[:label
[:text-input :qr-code]
```

I can either create another function or just unwrap the code, either ways I would be introducing some code duplication, but I don't mind either.

Reply:
So `qr-code` is some icon inside the input? If so my bad, sounds good.

Reply:
Yes, correct, thanks